requirements.txt

```diff
@@ -1,5 +1,6 @@
 Cython
 numpy
+matplotlib
```

Comment:
matplotlib is only used in the documentation as far as I can tell, it shouldn't be made a hard dependency